### PR TITLE
Global header

### PR DESCRIPTION
**Pull request by lancecummings**
Restructured the page templates to transition from the 2 existing templates (landing and default) to 3 templates (full-width, side-bar and guides). Full-width will encompass the home page and search results. Side-bar will be used for documentation pages. Guides will be used for content guidelines. 

Created new global header for full-width and side-bar templates. 

@muffinresearch

### CODE REVIEW

**Comment by muffinresearch** [comment]
@lancecummings I'll take a look at this tomorrow. 

FYI - Looks like you might be merging in a way that is replaying commits - if you have questions on git setup please let me know.

**Comment by lancecummings** [comment]
> @lancecummings I'll take a look at this tomorrow.
> 
> FYI - Looks like you might be merging in a way that is replaying commits - if you have questions on git setup please let me know.

Gah! Oh dear. I'll message you.

File: _assets/css/_fullwidth.scss [hunk 1]
```diff
@@ -41,14 +13,17 @@ Stylesheet: Landing Page Stylesheet
 
 .banner.intro {
   position: relative;
-  background: $white no-repeat center/cover;
+  background: transparent no-repeat center/cover;
```
[the remark this answers is not in the document]
Comment (lancecummings) [reply _assets/css/_fullwidth.scss]: This element could potentially have a background image assigned to it later

File: _assets/css/hamburgers/_hamburgers.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
+@charset "UTF-8";
+/*!
+ * Hamburgers
+ * @description Tasty CSS-animated hamburgers
+ * @author Jonathan Suh @jonsuh
+ * @site https://jonsuh.com/hamburgers
+ * @link https://github.com/jonsuh/hamburgers
```
[the remark this answers is not in the document]
Comment (lancecummings) [reply _assets/css/hamburgers/_hamburgers.scss]: Oh yeah! I thought I checked for that and didn't see it, but your right. I've swapped that.

File: _layouts/full-width.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                <div class="cell small-12 large-3">
+                    <h4>Twitter</h4>
+                    <p>For developers:
+                        <br /><a href="https://twitter.com/mozamo" class="twitter">@mozamo</a></p>
+                    <p>For end users:
+                        <br /><a href="https://twitter.com/rockyourfirefox" class="twitter">@rockyourfirefox</a></p>
+                </div>
+                <div class="cell small-12 large-4">
+                    <h4>More</h4>
+                    <p><a href="https://telegram.org" class="telegram">t.co/firefox</a></p>
```
[the remark this answers is not in the document]
Comment (lancecummings) [reply _layouts/full-width.html]: This is an existing module, so strictly speaking out of scope for this project. If I've got time left at the end I can revisit this.

File: _layouts/full-width.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        {{ content }}
+
+        <aside class="panel social">
+            <div class="grid-container grid-x grid-padding-x">
+                <div class="cell small-12">
+                    <h2>Connect With Us</h2>
+                </div>
+                <div class="cell small-12 large-3">
+                    <h4>Twitter</h4>
+                    <p>For developers:
```
[the remark this answers is not in the document]
Comment (lancecummings) [reply _layouts/full-width.html]: This is an existing module, so strictly speaking out of scope for this project. If I've got time left at the end I can revisit this.

File: _layouts/full-width.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                        <br /><a href="https://twitter.com/mozamo" class="twitter">@mozamo</a></p>
```
[the remark this answers is not in the document]
Comment (lancecummings) [reply _layouts/full-width.html]: This is an existing module, so strictly speaking out of scope for this project. If I've got time left at the end I can revisit this.

**Comment by lancecummings** [comment]
> Looks like there's a few changes needed here. Also I noticed a bunch of file permission change making some files executable which shouldn't be needed.
> 
> Just give me a mention when you need me to take another look.

Hey @muffinresearch, I'm not sure why there would be file permission changes. Is it on newly created files, or pre-existing ones? Let me know if there is anything I should be addressing specifically. I think other than that, we should be set for another look at this pull request. I also just sent you a question via Slack regarding my git setup.

File: package.json [hunk 6]
```diff
@@ -19,6 +19,7 @@
     "eslint": "5.16.0",
     "eslint-config-prettier": "4.3.0",
     "eslint-plugin-prettier": "3.1.0",
+    "hamburgers": "^1.1.3",
```
Comment (muffinresearch) [inline package.json]: This should be in dependencies as it's required as content in the site.

**Comment by lancecummings** [comment]
> I think the perms changes are coming in via icomoon, I presume it was updated and dropped in the tree, the executable permissions is probably how they're packaged.
> 
> This looks better, if you can fix the perms on the icomoon file and move hamburgers to deps instead devdeps this is ready to land.

@muffinresearch Great! Should be ready to go. Will wait for this to land and then rebase my search branch on master before creating the next pull request.

**Pull request merged**